Add termusic (#1044)
aharris88/awesome-cli: readme.md

- [radio-active](https://github.com/deep5050/radio-active) - Internet radio player with 40k+ stations.
- [mpvc](https://github.com/gmt4/mpvc) - Music player interfacing mpv.
- [TUISIC](https://github.com/Dark-Kernel/tuisic) - Login-free music streaming.

> - [termusic](https://github.com/tramhao/termusic) - Music Player written in Rust with an ability to download tracks.

Reviewer: ```suggestion
- [termusic](https://github.com/tramhao/termusic) - Music player in rust with download capabilities.
```
